### DISCUSSION TITLE
Whisper App

### DISCUSSION THREAD

**Discussion by Hannes1** [openai/whisper#464] [Show and tell]
Using whipser as a backend for [Writtan](https://www.writtan.com/)

Can check the download link for ios and android on the website

**Comment by ggerganov** [comment]
~If it is using Whisper, how come the latest releases of the app for iOS and Android are before the release date of Whisper?
Am I missing something?~
Edit: Nevermind, I missed that it is on the backend (thanks @nyadla-sys)

**Comment by nyadla-sys** [comment]
They might be using different ASR models to process on their cloud server, and they may have just enabled the whisper API on cloud.

**Comment by Hannes1** [comment]
@nyadla-sys yes, I was using wenet models previously, so I swapped them out for whipser

**Comment by nyadla-sys** [comment]
@Hannes1 You appear to be good in notebook writing; could you please look at the ones below and let me know?

I was able to convert from Hugging face whisper onnx to tflite(int8) model,however am not sure how to run the inference on this model
Could you please review and let me know if there is anything i am missing in onnx to tflite conversion
https://colab.research.google.com/github/usefulsensors/openai-whisper/blob/main/notebooks/whisper_to_onnx_tflite_int8.ipynb
or 
I was able to convert HF tf model to hybrid,however not able to generate quantized model 
https://colab.research.google.com/github/usefulsensors/openai-whisper/blob/main/notebooks/tflite_from_huggingface_whisper.ipynb

**Comment by vojto** [comment]
Interesting, I made a very simple wrapper for myself called [Whisper Memos](https://whispermemos.com/). 

How do you divide text into paragraphs? I'm using GPT-3 to do it, and it works pretty great - divides into paragraphs based on what I was talking about. However, it's pretty pricey to run GPT-3 on long notes.

**Comment by nyadla-sys** [comment]
Please feel free to download the openai/whisper-tiny tflite-based Android APP from Google App Store.
https://play.google.com/store/apps/details?id=com.whisper.android.tflitecpp

**Comment by stlukey** [comment]
Another solution I've used on iPhone as well as Mac and Windows is `whispercpp` (https://github.com/ggerganov/whisper.cpp). Uses a c++ runtimes for the model and accelerates with Arm's Neon, even being CPU only, drastically outperforms normal whisper using PyTorch on my M1 Mac. 

Pressumanly without a real GPU or other PyTorch based acceleration these significant performance increases extends to all modern Arm-based mobile devices.
